lib/frozen_record/test_helper.rb

```diff
@@ -20,20 +18,32 @@ def load_fixture(model_class, alternate_base_path)
         model_class.load_records(force: true)
       end
 
-      def unload_fixtures
+      def unload_fixture(model_class)
```

Comment:
You'd need to handle `unload_fixtures` being called without argument, in case some people depend on it.

Reply:
Definitely. It's already handed (https://github.com/byroot/frozen_record/pull/50/files#diff-760209d4b7ba5772608f810ae7977a39a84ffb8ec0142f2f47855ca236f69449R35). This diff is just being weird.

Reply:
Ah my bad, I misread.